crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# v0.4.0
```

Comment:
nit: there's a "queue" typo in the commit title.

Reply:
Done.